src/developer-testing/developer-testing.service.ts

```diff
@@ -0,0 +1,172 @@
+/* istanbul ignore file */
+
+import { Injectable } from '@nestjs/common';
+import { SubmittableExtrinsic } from '@polkadot/api-base/types';
+import { Keyring } from '@polkadot/keyring';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { Account, Identity } from '@polymeshassociation/polymesh-sdk/types';
+
+import { AccountsService } from '~/accounts/accounts.service';
+import { AppInternalError } from '~/common/errors';
+import { isNotNull } from '~/common/utils';
+import { CreateCddProviders } from '~/developer-testing/dto/create-admin.dto';
+import { CreateMockIdentityBatchDto } from '~/developer-testing/dto/create-mock-identity-batch';
+import { PolymeshService } from '~/polymesh/polymesh.service';
+import { SigningService } from '~/signing/services';
+
+const unitsPerPolyx = 1000000;
+const initialAdminPolyx = 100000000 * unitsPerPolyx;
```

Comment:
nit -> if we can provide an option to take the amount as input that will be really helpful. We can default to this value